index.md

```diff
@@ -108,10 +108,8 @@ readability and the required core data types.
 
 CFF `CITATION` files are made up of
 
-- exactly one message containing instructions on how to cite the software which
-the file is associated with; 
-- one or more references, containing at least `type`, `author`, and `title`
-information.
+- exactly one message containing instructions on how to cite the software which the file is associated with;
+- one or more works, containing at least `type`, `author`, and `title` information.
```

Comment:
I used the word `works` to make clear to the reader.

Reply:
I think this should be *references* throughout the specs and that therefore "Work Types" should be renamed to "Reference Types". This because I have the feeling that "reference" will be a more familiar term to users in the context of citations.

Reply:
I'm happy as long as we stay with only one term across the document.